### DISCUSSION TITLE
Questions about Mitigation Interventions

### DISCUSSION THREAD

**Discussion by schaupets** [neherlab/covid19_scenarios#641] [General]
I have three questions about the behaviour of the Mitigation Interventions: 

**1: Overlapping Interventions**
Per default, there are 2 interventions, with overlapping time
- 1. Intervention#1  20.march - 1.Sep = 50%
- 2. Intervention#2  29.march - 1.Sep = 60%

The interventions seem to be added somehow.
But simple addition of 50%+60% = 110% does not make sense!? 
I can add a third intervention with 40% which also has an effect, but not equivalent to 100% 

Question: how are the overlapping interventions added/merged/combined/used? 

**2: Intervention = 100%**
An Intervention of 100% should immediately stop new infections.
The model does this, but has some strange effects in the timeline, which I do not understand: 
Example for Germany:
Intervention from 1.July to 30.Sept = 100%  
- a) The number of infecctious decreases to zero only after 4 Weeks, altough latencies+infectious =  3+3 days . So I would expect a much faster decline after 6 days.
- b) the number of ICU and Severly ill decreases even slower and are greater Zero still after 2 Month.   With an hospital-stay = 3 days and ICU = 14 Days I would expect a decline to zero after about 3 weeks.

Question: What did I miss? 

**3: Interventions value for Effective-R0=1** 
My assumption for the algorithm is Effective-R0 = Average-R0 * (1- Intervention%)
To get an Effective-R0=1 for Germany this would require an 
Intervention-R1  =   1 - 1/Average-R0 = 1 - 1/3.6 = 72.22%  (ignoring seasonal variations) 

With Effective-R0=1 the number of infectious should remains constant. 
 The model does behave like this.

Question: Is my algorithm "Intervention-R1 = 1 - 1/Average-R0" correct? 

**Related to all questions:** Screenshot of my testing examples:
![COVID19-Scenarios-Mitigation-Examples-01](https://user-images.githubusercontent.com/64103927/80588931-88abd700-8a19-11ea-81f5-d18ee672eee1.jpg)

Thats what I found:  https://covid19-scenarios.org/about
Transmission reduction: The tool allows one to explore temporal variation in the reduction of transmission by infection control measures

**Comment by AxiaTim** [comment]
Hi, 

I had the same question as your first one. I found this in the Questions, discussions thread:

https://github.com/neherlab/covid19_scenarios/discussions/18#discussioncomment-4033


> 
> an overlap of a 40% and 60% measure would results in a reduction of transmission by (1-0.4)*(1-0.6)
>
